server/database/sequelize.ts

```diff
@@ -32,3 +34,27 @@ export const sequelize = new Sequelize(url, {
     idle: 10000,
   },
 });
+
+export const umzug = new Umzug({
+  migrations: {
+    glob: ["migrations/*.js", { cwd: path.resolve("server") }],
+    resolve: ({ name, path, context }) => {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const migration = require(path as string);
+      return {
+        name,
+        up: async () => migration.up(context, Sequelize),
+        down: async () => migration.down(context, Sequelize),
+      };
+    },
+  },
+  context: sequelize.getQueryInterface(),
+  storage: new SequelizeStorage({ sequelize }),
+  logger: {
+    warn: (msg) => Logger.warn("database", msg),
+    error: (msg) =>
+      Logger.error("database", new Error("Migration failed!"), msg),
```

Comment:
```suggestion
      Logger.error(msg, new Error("Database migration failed")),
```

Reply:
Needs to be stringified...
```suggestion
      Logger.error(JSON.stringify(msg), new Error("Database migration failed")),
```

Reply:
What kind of object is `msg` then? I assumed it was a string…

Reply:
In that case best passed as the third `extra` argument, first just has to be a repeat of "Database migration failed"